moped-editor/src/layouts/DashboardLayout/NavBar/NavigationSearchQueryConf.js

```diff
@@ -37,6 +37,24 @@ export const NavigationSearchQueryConf = {
       },
       type: "String",
     },
+    project_name_secondary: {
+      searchable: true,
+      search: {
+        operator: "_ilike",
+        quoted: true,
+        envelope: "%{VALUE}%",
+      },
+      type: "String",
+    },
+    project_name_full: {
```

Comment:
Since we are checking if the search string is anywhere in all three project name fields, I wonder if we should remove the other two and keep `project_name_full` since it is the combo of the other two. It works the same either way. Really just thinking out loud.

Reply:
makes sense to remove the other two imo!

Reply:
makes sense, thanks for thinking of that!